flightlib/cmake/eigen_download.cmake

```diff
@@ -5,7 +5,7 @@ project(eigen-external)
 include(ExternalProject)
 ExternalProject_Add(eigen
   GIT_REPOSITORY    https://gitlab.com/libeigen/eigen.git
-  GIT_TAG           master
+  GIT_TAG           3.3.4
```

Comment:
I don't think we should fix the Eigen version.

Reply:
the current version of Eigen (3.3.8) has bugs. What would you recommend instead?